src/fswAlgorithms/effectorInterfaces/thrusterPlatformReference/thrusterPlatformReference.c

```diff
@@ -65,6 +65,13 @@ void Reset_thrusterPlatformReference(ThrusterPlatformReferenceConfig *configData
     else {
         configData->momentumDumping = No;
     }
+
+    /*! set the RW momentum integral to zero */
+    for (int i=0; i<3; ++i) {
```

Comment:
:large_orange_diamond: Is there a case in which this would ever not be a 3 vector? If not, would `v3SetZero` work well here?

Reply:
Absolutely. I'd forgotten we had that function. Fixed it.